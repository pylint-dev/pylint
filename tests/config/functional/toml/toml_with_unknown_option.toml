# Check that we handle unknown options correctly

[tool.pylint]
unknown-option = "yes"
jobs = "10"
another-unknown-option = "yes"
