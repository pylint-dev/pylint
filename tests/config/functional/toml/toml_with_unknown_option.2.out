************* Module {abspath}
{relpath}:1:0: E0015: Unrecognized option found: unknown-option, another-unknown-option (unrecognized-option)
